Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -88,14 +109,140 @@ or
 
     "path": "`name.with.dots`"
 
-xify_front
-----------
+If the attribute value is an array the masking is applied to all the
+array elements.
+
+### Example
+
+This following will replace name by a "XXXX" masked string
+
+    {
+      "type": "xify_front",
+      "path": ".name",
+      "unmaskedLength": 2
+    }
```

Comment:
This should rather be using fenced code (in triple backticks, with the first four spaces removed) and JSON highlighting

```
```json
{
  ...
}
```

Reply:
done